### PR TITLE
Replace IRC info with Slack

### PR DESCRIPTION
**Pull request by nickygerritsen**
As a PR so we can discuss if we should modify what I typed.

### CODE REVIEW

File: website/slack.shtml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <a href="https://domjudge-org.slack.com/" target="_blank">https://domjudge-org.slack.com/</a>.
+    To join as a new member, you can use
+    <a href="/slack/join" target="_blank">this page to sign up</a>.
+</p>
+<p>
+    Once you have joined the Slack Workspace, feel free to ask any DOMjudge-related
+    questions you might have. Please do note that it may sometimes take a bit longer
+    than a few minutes to get a response.
+
+    Because Slack has a limit of 10.000 messages that it will keep on the free plan,
```
Comment (meisterT) [inline website/slack.shtml]: Actually is that archiving GDPR compliant if we do it as a service?

File: website/header.shtml [hunk 2]
```diff
@@ -29,6 +29,9 @@
     <li class="nav-item"><a
       class="nav-link<!--#if expr="v('DOCUMENT_NAME') = 'about.shtml'" --> active<!--#endif-->"
       href="about"><i class="far fa-lightbulb"></i> About</a></li>
+    <li class="nav-item"><a
+      class="nav-link<!--#if expr="v('DOCUMENT_NAME') = 'slack.shtml'" --> active<!--#endif-->"
+      href="slack"><i class="fab fa-slack"></i> Slack</a></li>
```
Comment (thijskh) [inline website/header.shtml]: would call it 'chat' or something more generic than the brand of chat we use. can keep the icon as a hint to which brand we're using today.

File: website/slack.shtml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<!--#set var="dj_page_title" value="Slack" --><!--#include virtual="/header.shtml" -->
+
+<p>
+    DOMjudge offers a Slack Workspace where developers and users can discuss
+    any questions, comments, ideas and other DOMjudge related things they want.
```
Comment (thijskh) [inline website/slack.shtml]: add something that this is about chatting

File: website/slack.shtml (hunk 1 above)
Comment (thijskh) [reply website/slack.shtml]: You could say that the ground is consent since users are free to use this, and informed on this page before they join.

Comment (nickygerritsen) [reply website/slack.shtml]: We might also pin a message in #general telling them that we do it?

File: website/header.shtml (hunk 2 above)
Comment (nickygerritsen) [reply website/header.shtml]: 👍 . Also if we upgrade font-awesome we get the new Slack icon

Comment (nickygerritsen) [reply website/header.shtml]: Ah the upgrading needs to be done in demoweb

File: website/slack.shtml (hunk 1 above)
Comment (meisterT) [reply website/slack.shtml]: There's not only consent, but also right to be forgotten: https://gdpr-info.eu/art-17-gdpr/

Comment (nickygerritsen) [reply website/slack.shtml]: Would `delete from messages where user_id = <theuserid>` (and deleting from `users`) be enough for this, if we get a request to remove the logs? We then miss other people quoting the messages and/or highlighting the user...

Comment (eldering) [reply website/slack.shtml]: Does the GDPR even apply to us as long we don't have the stichting DOMjudge foundation yet?

Comment (meisterT) [reply website/slack.shtml]: 🤷‍♂️ IANAL, but I also don't need that long of a history.

Comment (thijskh) [reply website/slack.shtml]: Personally, I see chat as something ephemeral similar to real world chat.

**Pull request merged**